index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+    <main class="container">
+        <header class="main-header">
+            <p class="dev-name">Mehsu Amare</p>
+            <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M11 15C11 14.4477 11.4477 14 12 14H28C28.5523 14 29 14.4477 29 15C29 15.5523 28.5523 16 28 16H12C11.4477 16 11 15.5523 11 15ZM11 20C11 19.4477 11.4477 19 12 19L28 19C28.5523 19 29 19.4477 29 20C29 20.5523 28.5523 21 28 21L12 21C11.4477 21 11 20.5523 11 20ZM12 24C11.4477 24 11 24.4477 11 25C11 25.5523 11.4477 26 12 26H28C28.5523 26 29 25.5523 29 25C29 24.4477 28.5523 24 28 24H12Z"
+                    fill="white" />
+            </svg>
+        </header>
+        <section class="headline">
+            <h1 class="title">Hey There. I'm Melashu</h1>
+            <p class="supporting">I’am a Software Developer</p>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <a href="#"><img src="image/git.png" alt="github"></a>
+                <a href="#"><img src="image/linkedin.png" alt="linkedin"></a>
+                <a href="#"><img src="image/take.png" alt="github"></a>
+                <a href="#"><img src="image/twitter.png" alt="twitter"></a>
+                <a href="#"><img src="image/medium.png" alt="medium"></a>
+            </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)